### PR TITLE
DDP-6303 update header and add i18n for header and footer for PanCan

### PR DESCRIPTION
**Pull request by aglushko-broad**
- replaced toolkit header with custom header since we can't use it here in PanCan
- copied based styles for layout and hamburger menu header from Osteo
- used redesigned components and update i18n for them
- add i18n for header and footer

### CODE REVIEW

File: ddp-workspace/projects/ddp-pancan/src/variables.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$header-height: 6rem;
+$footer-height: 14.94rem;
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-pancan/src/variables.scss:2]: #minor
Do we need such an accuracy there ?

**Review by alextanlan — APPROVED** [review]

File: ddp-workspace/projects/ddp-pancan/src/variables.scss (hunk 1 above)
Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-pancan/src/variables.scss:2]: why not? any issues about that?

Comment (alextanlan) [reply ddp-workspace/projects/ddp-pancan/src/variables.scss:2]: Not an issue, just a strange value, as for me.

Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-pancan/src/variables.scss:2]: I calculated it based on mockup sizes and keep 2 numbers after the dot. I hope it's more clear now.

**Pull request merged**